lib/guessingGame.js

```diff
@@ -0,0 +1,29 @@
+var pry = require('pryjs');
+
+
+function GuessingGame(answer) {
+  this.answer = answer;
+  this.guess = function (guess) {
+    this.current_guess = guess;
+    if(guess === this.answer){
+        return 'correct';
+        }
+    else if (guess > this.answer){
+      return 'high';
+      }
+    else if (guess < this.answer){
+      return 'low';
+      }
+    }
+  this.isSolved = function(){
+
+    if(this.current_guess === this.answer){
```

Comment:
You may get an error if you try to call `isSolved` before ever guessing since you are setting `this.current_guess` for the first time in the guess function.